### DISCUSSION TITLE
Components inside Drawer no longer work after updating to from "AntDesign" Version="0.12.7" to "AntDesign" Version="0.14.2"

### DISCUSSION THREAD

**Discussion by itblogpros** [ant-design-blazor/ant-design-blazor#3101] [General]
Hi everyone, hope this is ok to post here. 

I have multiple drawers in my web app containing drop-down boxes, calendars and text inputs. Since I updated to 0.14.2, the dropdowns and calendars no longer work. However, the text fields are all ok. The data populates each of these components as expected (they pull data from the DB and make the items appear correctly.) However, when I try to click on a drop-down item, the drop-down menu does not respond to the change, neither does the calendar etc.

I have looked at all of the docs, and I appear to be doing everything correctly, but this problem persists. As a test, I rendered these components within the row, and everything works as expected; it is only once it is inside of a drawer that it stops working. Everything works as expected in "AntDesign" Version="0.12.7".

Any suggestions?

**Comment by zxyao145** [comment]
Same as #3092 and #3102, it will be repaired soon.
